src/nodes/Table/TableView.vue

```diff
@@ -22,20 +22,41 @@
 			</NcActionButton>
 		</NcActions>
 		<NodeViewContent class="content" as="table" />
+		<NcButton v-if="isEditable"
+			class="table-add-column"
+			:aria-label="t('text', 'Add column after')"
+			:title="t('text', 'Add column after')"
+			@click="addColumnAfter">
+			<template #icon>
+				<TableAddColumnAfter />
+			</template>
+		</NcButton>
+		<NcButton v-if="isEditable"
+			class="table-add-row"
+			:aria-label="t('text', 'Add row after')"
```

Comment:
```suggestion
			:aria-label="t('text', 'Add row below')"
```

Maybe better language-wise?